api-reference/beta/resources/alert.md

```diff
@@ -0,0 +1,120 @@
+# alert resource type
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Alerts represent potential security issues within a customer's tenant that have been identified by Microsoft or partner security solutions. Alerts can be used to unify and streamline alert management across all integrated solutions. See sample queries in [Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer).
+
+For Public Preview (April 2018), alerts can be retrieved from the following Providers: **Azure Security Center**, and **Azure Active Directory Identity Protection**. Additional Alert Providers will integrate in the following months.
+
+## Methods
+
+| Method   | Return Type|Description|
+|:---------------|:--------|:----------|
+|[Get alert](../api/alert_get.md) | [alert](alert.md) |Read properties and relationships of alert object.|
+|[Update](../api/alert_update.md) | [alert](alert.md) |Update alert object. |
+|[List alerts](../api/alert_list.md) | [alert](alert.md) collection |Get a alert object collection.|
+
+## Properties
+
+### ( \* = Mandatory Field )
+
+| Property   | Type|Description|
+|:---------------|:--------|:----------|
+|activityGroupStates|[activityGroupState](activitygroupstate.md) collection|Threat Intelligence activity group(s) that this alert is attributed to.|
+|applicationStates|[applicationSecurityState](applicationsecuritystate.md) collection|Security-related stateful information generated by the provider about the application(s) related to this alert. |
+|assignedTo|String|Name of the analyst the alert is assigned to for triage, investigation, or remediation ([Supports Update](../api/alert_update.md)).|
+|azureSubscriptionId|String|Azure subscription ID, present if this alert is related to an Azure resource.|

```

Comment:
"Supports" shouldn't be part of the link text, and also doesn't need to be capped.